code/Data exploration.R

```diff
@@ -175,16 +200,46 @@ data %>% filter(is.na(TEMPERATURE))
 data %>% filter(is.na(INITIAL_FORECAST))
 data %>% filter(is.na(FINAL_FORECAST))
 
-############################
-##DERIVE ADDITIONAL FIELDS##
-############################
+################################
+##Aggregate the data to hourly##
+################################
 
-##convert DATETIME to date -- no longer required, reading in as right field on auto
-##data$DATETIME = ymd_hms(data$DATETIME);
+#Since the forecast made by models will be hourly
 
 ##create year/month/day/hour/minute fields
-data <- data %>% mutate(YEAR=year(DATETIME), MONTH=month(DATETIME),DAY=day(DATETIME));
-data <- data %>% mutate(HOUR=hour(DATETIME), MINUTE=minute(DATETIME));
+data <- data %>% mutate(YEAR=year(DATETIME), 
+                        MONTH=month(DATETIME),
+                        DAY=day(DATETIME),
+                        HOUR=hour(DATETIME), 
+                        MINUTE=minute(DATETIME));
+
+##Create DATE and MONTHDATE field for joining/plotting
+data$DATE <- as.Date(paste(year(data$DATETIME), 
+                           month(data$DATETIME),
+                           day(data$DATETIME), sep = "-"))
+data$YEARMONTH<- as.Date(paste(year(data$DATETIME), 
+                               month(data$DATETIME),
+                               '01', sep = "-"))
+
+#Aggregate the fields
+#Agg total demand via sum
+#Agg initial/final forecast via sum
+#Agg temperature by mean (thought of using max/min, but took the unbiased approach)
+data <- data %>%
+  group_by(DATE,
```

Comment:
is it possible to just to create a date_time field that specifies up to hour instead?

Reply:
can be done with unite, will make the change